test/slicing/test_slicing.py

```diff
@@ -0,0 +1,172 @@
+import random
+import unittest
+from functools import partial
+
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
+
+from snorkel.mtl.data import MultitaskDataLoader, MultitaskDataset
```

Comment:
Let's do some thinking later re: simplification (add a task please). Too many intra-snorkel deps for a unit test

Reply:
Yeah, this is more of an integration test. Just converted the minimal script I was using. Agreed that we should add some finer grained tests. Added to the tracker.